<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>Push me</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumber1"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton2">
      <property name="text">
       <string>No me!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber2"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton3">
      <property name="text">
       <string>Meee!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber3"/>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>198</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Connect">
    <property name="title">
     <string>Connect to...</string>
    </property>
   </widget>
   <addaction name="menu_Connect"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
